packages/playground/_stories/fiori/DynamicPage/DynamicPage.stories.ts

```diff
@@ -859,6 +858,36 @@ Basic.args = {
     </ui5-bar>`,
 
 };
+
+Basic.decorators = [
+    (story) => html`
+    ${story()}
+<script>
+    const page = document.getElementById("page");
+    const scrollDownStep = 500;
+
+    function getScrollContainer() {
+        return page.shadowRoot.querySelector(".ui5-dynamic-page-scroll-container");
+    }
+
+    document.getElementById("toggleFooterBtn").addEventListener("click", () => {
```

Comment:
This script tag throws an error in the console and prevents modifying the example using the storybook controls. If I remember correctly we talk that they are not needed, as they were only added for testing purpose.